app/settings.ts

```diff
@@ -217,6 +218,7 @@ export class LocalSettings extends Settings<LocalSettingsData> {
     if (!settings.version || settings.version < 7) {
       log.verbose('Found settings version 6, migrating to version 7')
       newSettings.runAppAtSystemStart = true
+      newSettings.startAppMinimized = false
```

Comment:
You should increase the `VERSION` of settings whenever they change, since some users might already be on this version and this migration won't run for them.